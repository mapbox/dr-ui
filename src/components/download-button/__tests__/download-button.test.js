import renderer from 'react-test-renderer';
import { testCases } from './download-button-test-cases.js';

describe('DownloadButton', () => {
  describe(testCases.basic.description, () => {
    let testCase;
    let wrapper;
    let tree;

    beforeEach(() => {
      testCase = testCases.basic;
      wrapper = renderer.create(testCase.element);
      tree = wrapper.toJSON();
    });

    test('renders as expected', () => {
      expect(tree).toMatchSnapshot();
    });
  });
});
