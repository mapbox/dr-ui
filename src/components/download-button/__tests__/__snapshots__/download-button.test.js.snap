// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DownloadButton Default DownloadButton renders as expected 1`] = `
<a
  className="btn btn--blue round-full py12 px24 txt-s"
  href="../files/airports.csv"
>
  <span
    className="flex-parent flex-parent--center-cross"
  >
    <span
      className="flex-child mr3"
    >
      <svg
        className="events-none icon"
        focusable="false"
        role="presentation"
        style={
          Object {
            "height": 18,
            "width": 18,
          }
        }
      >
        <use
          xlinkHref="#icon-arrow-down"
          xmlnsXlink="http://www.w3.org/1999/xlink"
        />
      </svg>
    </span>
    <span
      className="flex-child"
    >
      Download 
      CSV
    </span>
  </span>
</a>
`;
